contributing/community/releases.rst

```diff
@@ -150,6 +150,10 @@ of Symfony to the next one.
 Whenever keeping backward compatibility is not possible, the feature, the
 enhancement or the bug fix will be scheduled for the next major version.
 
+However, features marked with the ``@experimental`` tag are **not** part of the
+backward compatibility promise and their APIs can change from one minor version
+to the next. The changelog must explain the changes and how to upgrade.
```

Comment:
s/must/should/? It's not always worth the burden (eg we didn't follow that "must" for LDAP, and it was fine?)